codespell_lib/data/dictionary.txt

```diff
@@ -11577,6 +11580,7 @@ ediges->edges
 editedt->edited
 editiing->editing
 editoro->editor
+editot->editor
```

Comment:
Please also add `editots->editors` and squash all commits in to one (don't wor

Reply:
Done!